modal/sandbox.py

```diff
@@ -323,6 +330,63 @@ async def _load(self: _Sandbox, resolver: Resolver, _existing_object_id: Optiona
 
         return _Sandbox._from_loader(_load, "Sandbox()", deps=_deps)
 
+    @staticmethod
+    async def create(
+        *entrypoint_args: str,
```

Comment:
Should we make the following args keyword-only?

Reply:
yeah good point

Reply:
actually believe they are keyword-only because of the * automatically (i just learned something new about python!)

Reply:
oh cool, makes sense!